code/game/machinery/airlock_cycle_control.dm

```diff
@@ -129,12 +129,12 @@
 	qdel(wires)
 	wires = null
 	cut_links()
-	SSair.stop_processing_machine(src)
```

Comment:
that proc is there for a reason y'know

Reply:
Is that a really recent addition? because I didn't touch that and I updated my codebase a few days ago.

Reply:
yeah, just got added today

Reply:
Yeah I'll update my code and then remake my changes then